apps/heft/src/startWithVersionSelector.ts

```diff
@@ -50,6 +50,11 @@ function tryStartLocalHeft(): boolean {
     console.log('(Bypassing the Heft version selector because "--unmanaged" was specified.)');
     console.log();
     return false;
+  } else {
+    // The unmanaged flag could be undiscoverable if it's not in their locally installed version
+    console.log(
+      'Searching for a locally installed version of Heft. Use the --unmanaged flag if you want to avoid this'
```

Comment:
@iclanton What's your opinion about this? I was trying to minimize the extra noise that we print on the console. If we weren't concerned about minimizing exports, I would have preferred to use a dim terminal color to make these lines less noticeable.

```
// The unmanaged flag could be undiscoverable if it's not in their locally installed version
```
Heft is still in early preview, so by the time it's released we probably won't care too much about ancient versions people might have installed. (?)

Reply:
Yeah I think --debug is a reasonable flag for this, thanks for the feedback